src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1765,6 +1768,17 @@ static bool processOnChangeConditionForUpdateContext
       }
     }
   }
+  if (covered)
+  {
+    for (unsigned int jx = 0; jx < attrL.size(); jx++)
+    {
+      const std::string& _type = DEFAULT_ATTR_NULL_TYPE;
+      const std::string& _value = "null";
+      std::string attrName = attrL[jx];
+
+      cer.entity.attributeVector.push_back(new ContextAttribute(attrName, _type, _value));
+    }
+  }
```

Comment:
Note that `"null"` is not the same that `null` and we want the later one here. Thus, this should be changed to something like this:

```suggestion
  if (covered)
  {
    for (unsigned int ix = 0; ix < attrL.size(); ix++)
    {
      std::string attrName = attrL[ix];
      ContextAttribute* caP = new ContextAttribute(attrName, DEFAULT_ATTR_NULL_TYPE, "");
      caP->valueType = orion::ValueTypeNull;
      cer.entity.attributeVector.push_back(caP);
    }
  }
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4068/commits/feff393ec16f6e7f09fd4a04dd94fb97d4f547e7